src/gui/systray.cpp

```diff
@@ -93,6 +102,8 @@ void Systray::create()
             _trayEngine->rootContext()->setContextProperty("activityModel", UserModel::instance()->currentActivityModel());
         }
         _trayEngine->load(QStringLiteral("qrc:/qml/src/gui/tray/Window.qml"));
+
+        _trayEngine->rootContext()->setContextProperty("appNameGUI", Theme::instance()->appNameGUI());
```

Comment:
Don't change it now. FYI we'll have to change that later on, context properties are deprecated, that's why we got Nico's patches to switch to QML singletons. That can wait.